book.bib

```diff
@@ -548,6 +548,21 @@ @Manual{R-TreeSummarizedExperiment
   note =	 {R package version 1.6.2},
 }
 
+@Manual{R-MultiAssayExperiment,
```

Comment:
We should not use "-" in bibtex keys. It may cause problems sometimes. 

-> Replace "-" in bibtext keys with "_" or "".

Reply:
(in any reference)

Reply:
I see lot's of   "-" in the bibtext within the document. Should all be removed?

Reply:
It only matters when it is in the key, meaning 

@article{name-with-dash

@manual{name-with-dash

etc.

In article titles etc. it is ok.

Yes, for all keys it would be good to change, and probably easiest to do for all at once.

On the other hand, if we do not have problems with it now in current setup I think we can leave this later in the end. Let's keep in mind anyway.